directus-remix/index.md

```diff
@@ -0,0 +1,331 @@
+---
+title: 'Build a Website With Remix and the Directus JavaScript SDK'
+description: 'A starter guide on how to build a website using Remix framework and Directus as a headless CMS'
+author:
+  name: 'Trust Jamin'
+  avatar_file_name: 'jamin.png'
+---
+
+## Introduction
+
+[Remix](https://remix.run/) is a React.js framework that  helps you deliver websites that are fast, slick, and have resilient user experience. In this tutorial, you will learn how to build a website using Directus as a headless CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
```

Comment:
```suggestion
[Remix](https://remix.run/) is a React.js framework that provides many conveniences useful when building websites. In this tutorial, you will learn how to build a website using Directus as a Headless CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
```

This suggestion removes overpromoting or selling Remix and just explains what it is plainly.